_historical_review/core-tasks/tests/test_neo4j_entity_service.py

```
@@ -0,0 +1,510 @@
+        to_type="Entity",
+        relationship_type=RelationshipType.MENTIONS,
+        properties={"confidence": 0.95, "count": 3}
+    )
+
+    assert result is True
+    call_args = mock_connection.execute_query.call_args
+    query = call_args[0][0]
+
+    assert "MATCH" in query
+    assert "MERGE" in query or "CREATE" in query
+    assert "MENTIONS" in query or relationship_type.value in query
```

Ruff (0.14.13)

Undefined name `relationship_type`

(F821)